index.html

```diff
@@ -54,7 +54,7 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
                   max="99"
                   class="form-control"
                   placeholder="Age"
-                />
+                />hhhhm,.m jj
               </div>
               <div class="form-group">
                 <p>Which option best describes your current role?</p>
```

Comment:
# Wonderful Great effory
**ACHIVEMENTS**

> PAGE BUILT EFFECTIVELY

> FORM SETUP

# REQUIRED CHANGES'

1.  I noticed  stray text in your code  Kindly wrap it in <p> tag  if it is intended  as it doesn't really appear professional
2. I suggest you also style the test using margin to avoid  it clashing with your form
3. 

- 

>